.github/workflows/slack-deployment-message.yml

```
@@ -97,10 +99,16 @@ jobs:
             slack_channel='C033EL7UMS6'
           fi
 
+          if [ -z $commit_pr_number ]; then
+            pull_request_slack_message_part="without PR"
```

```suggestion
            pull_request_slack_message_part="(not part of an open PR)"
```

This made me think: even if the commit message is a conventional commit, isn't the #XXXX part the GH issue, not the PR, so if we use that, and build a URL from it that points to a pull request, we'll hit a 404 or similar, won't we?

---

Yes, that is correct. That is why I wasn't so sure about the last test case from the description: 
```
when the PR is not created and the commit is in a format with (#000) it will take the 000 and use it as a PR id number
```

That was there before and I am guessing that updating this will affect every repo CI. What should we do?

---

Just for transparency:

Agreed with Petra to not include the PR link if the PR doesn't exist. Later we can come back and make it work with the issues if that was the real intention before.

---

That sounds good to me 👌 

What do you think about the text suggestion to say "(not part of an open PR)" or "(not in a PR)" instead of "without a PR"?

I could also imagine simply not saying anything if that's the case.

---

Yeah, we can go with `not part of an open PR`. Didn't notice the suggestion you made earlier.